components/zenFunctions/client/fn_zen_createSpawner.sqf

```diff
@@ -1,5 +1,7 @@
 #include "macros.hpp"
-
+/*
+Created By: Gibbs
```

Comment:
I'm 100% up for you getting attribution but if you don't indent your multiline comments I will find some way to revoke your coding license *i swear to god*

```suggestion
    Created By: Gibbs
```